README.md

```diff
@@ -115,10 +115,11 @@ just the text of your transcript
 
 ### Getting captions output
 
-You can also get captions output from the SDK:
+You can also get captions output from the SDK. We offer both SRT and VTT caption formats.
+If you submitted your job as a multichannel then you must also provide a `channel_id` to be captioned:
```

Comment:
I think we refer to it as speaker channels.

Speaker Channels = 1 speaker per channel
Multi Channel = generic term for audio with multiple channels (example: stereo audio)

```suggestion
If you submitted your job as a speaker channel then you must also provide a `channel_id` to be captioned:
```